src/libponyrt/sched/scheduler.c

```diff
@@ -1411,7 +1411,7 @@ void ponyint_sched_mute(pony_ctx_t* ctx, pony_actor_t* sender, pony_actor_t* rec
     int64_t old_mset_alloc_size = ponyint_muteset_alloc_size(&mref->value);
 #endif
     ponyint_muteset_putindex(&mref->value, sender, index2);
-    atomic_fetch_add_explicit(&sender->muted, 1, memory_order_acq_rel);
+    sender->muted--;
```

Comment:
The old version was doing fetch_add

```suggestion
    sender->muted++;
```